posthog/models/feature_flag/feature_flag.py

```diff
@@ -260,6 +260,7 @@ def get_cohort_ids(
         self,
         using_database: str = "default",
         seen_cohorts_cache: Optional[Dict[str, Cohort]] = None,
+        sort_by_creation_order=False,
```

Comment:
one test for updating the cohort later on (instead of adding connections on creation) would be nice, just so we have a test case for this knowledge, so if someone in the future tries to sort by ids, this test will fail and explain what's wrong 😅

Reply:
I added:
- a test for the topological sort in `test_cohorts_sorted_topologically` (_test_feature_flag_utils.py_)
- a test for updating the cohorts _later on_: `test_copy_feature_flag_cohort_nonexistent_in_destination_2` (test_organization_feature_flag.py_) - this also tests for the presence of the whole cohort "tree" in the new flag